### DISCUSSION TITLE
In config file, name must be the same as aedt file name

### DISCUSSION THREAD

**Discussion by hui-zhou-a** [ansys/aedt-testing#223] [General]
In config file, name must be the same as aedt file name. Otherwise, it cannot find the results. 

<img width="256" alt="image" src="https://user-images.githubusercontent.com/27995305/224023109-cbaf72c9-1d4c-456c-947f-a1e7e2d3546c.png">

**Comment by beliaev-maksim** [comment]
I would address this as a bug

can be caught on the settings validation
